src/redist/targets/packaging/windows/clisdk/sdkplaceholder.wxs

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?include "Variables.wxi" ?>
+  <Product Id="*" Name="$(var.ProductName) from Visual Studio" Language="$(var.ProductLanguage)" Version="$(var.ProductVersion)" Manufacturer="$(var.Manufacturer)" UpgradeCode="$(var.UpgradeCode)">
+    <Package Compressed="yes" InstallScope="perMachine" InstallerVersion="200" />
+
+    <Condition Message="$(var.ProductName) must be installed as part of a coordinated SDK installation.">
+      Installed OR ALLOWMSIINSTALL
+    </Condition>
+
+    <Condition Message="$(var.ProductName) was installed with Visual Studio. If you wish to uninstall, use the Visual Studio Installer and select: &apos;Modify&apos;">
```

Comment:
This seems like something that should be localized, but we only currently have the infra for localized bundles. I don't know if we can do anything about this now, though. 😕